networking/ovn_kubernetes_network_provider/using-an-egress-router-ovn.adoc

```diff
@@ -0,0 +1,14 @@
+[id="using-an-egress-router-cni"]
+= Considerations for the use of an egress router pod
+include::modules/common-attributes.adoc[]
+:context: using-an-egress-router-cni
+
+toc::[]
+
+include::modules/nw-egress-router-about.adoc[leveloffset=+1]
+
+[id="using-an-egress-router-additional-resources-cni"]
```

Comment:
I'd probably have `additional-resources` at the end instead of `cni`.

Reply:
ditto!